src/components/carbon-provider/carbon-provider.component.js

```diff
@@ -3,10 +3,14 @@ import PropTypes from "prop-types";
 import { ThemeProvider } from "styled-components";
 
 import mintTheme from "../../style/themes/mint";
+import CarbonScopedTokensProvider from "../../style/design-tokens/carbon-scoped-token-provider";
 
 const CarbonProvider = ({ children, theme = mintTheme }) => (
-  <ThemeProvider theme={theme}>{children}</ThemeProvider>
+  <ThemeProvider theme={theme}>
+    <CarbonScopedTokensProvider>{children}</CarbonScopedTokensProvider>
```

Comment:
We need to pass the theme into the `CarbonScopedTokensProvider` as well don't we?

Reply:
No, we don't need to - it inherits theme from `ThemeProvider`

Reply:
Ah yeah, Monday morning question 😅